debian/grub/acrn.cfg

```diff
@@ -1,20 +1,36 @@
-# No override variables here any more. To add any custom kernel parameters
-# for ACRN Service VM boot, please adapt your scenario configuration
-# accordingly!
-
 # Make booting into ACRN the default. Set the following variable to 0 or 1
 # explicitly to avoid the message warning from being printed.
 #
 #ACRN_OVERRIDE_GRUB_DEFAULT=
 #
+
+if [ "x${GRUB_DISTRIBUTOR}" = "x" ] ; then
+  OS=GNU/Linux
+else
+  case ${GRUB_DISTRIBUTOR} in
+    Ubuntu|Kubuntu)
+      OS="${GRUB_DISTRIBUTOR}"
+      ;;
+    *)
+      OS="${GRUB_DISTRIBUTOR} GNU/Linux"
+      ;;
+  esac
+fi
```

Comment:
How about using a single `case` statement to cover all cases? An empty string can be matched using `"")`.

Reply:
I took this from what I found in /etc/grub.d/10_linux on Debian Bullseye. But, of course, we can simplify that.